### ISSUE TITLE
Why do I have to build the widget myself?

### ISSUE THREAD

**Issue by MattiSG** [sgmap/ludwig#20]
If I have to instantiate it, why do I need to give it the configuration object, build it and copy the `bundle.js` to my app?

**Comment by hoshin** [comment]
As of now, the widget is bundled / published as part of the module (in the /dist dir). The widget in itself needs to be instantiated with an appropriate configuration. 

If the issue is that you wonder if you have to rebuild the widget at each configuration change, the answer is "not anymore". You just need to modify the app that uses the widget so that it gives it a different configuration object.

Basically, to have a useable widget, you can copy the `/dist/bundle.js` file and use it "as is". You still have to instantiate it with some configuration (detailed in the README file) so that the widget knows what repo to work on and which templates to use, though.

**Comment by MattiSG** [comment]
:+1:
